app/views/shared/_notifications.html.slim

```diff
@@ -4,4 +4,4 @@
         locals: { messages: value, alert: key, float: flash[:float], klass: "" }
 
 = render template: 'shared/_notification.html.slim',
-    locals: { messages: "Could not connect with Velum API. Please, try #{link_to "reloading the session", root_path}.".html_safe, alert: "alert", klass: "connection-failed-alert", float: false }
+    locals: { messages: "Could not connect with Velum API. If you're bootstrapping for the first time, this is expected due to an update of our certificate. Please, try #{link_to "reloading the session", root_path}.".html_safe, alert: "alert", klass: "connection-failed-alert", float: false }
```

Comment:
This sounds bad for me again. Instead presenting this message as error, we should ask user kindly to refresh his browser due to certs regeneration. If we can't handle this in other way, this error should stay as exception. It should be also print on more neutral background yellow or even green. Everything in this state makes bad impression during installation and it is too intrusive and from user experience feels really bad.

Reply:
We can't, that I know of, make a determination between "certificate changed", and "you unplugged your ethernet cable". Both look identical to us, so the same error will be shown. Giving the user an indication as to why it may have occurred for the case we know everyone will hit is a good thing...